.github/labeler.yml

```diff
@@ -1,7 +1,11 @@
 ---
 
+Style:
+  - app/**/*.sccs
```

Comment:
I'm guessing you're not wanting to label old internal repo files from the Source Code Control System as Style?

Maybe label `*css` and `*.scss?

Would a label of 'Design' make more sense as well? In which case I'd than move the ERB files into 'Design' also?

Reply:
Dead right sccs is an old Unix "Source Code Control System" file,